sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -341,6 +341,12 @@ case class LoadDataCommand(
       } else {
         val uri = new URI(path)
         if (uri.getScheme() != null && uri.getAuthority() != null) {
+          val hadoopConf = sparkSession.sessionState.newHadoopConf()
```

Comment:
You can probably lift this out of the if-else and use it in the other branch too

Reply:
+1